src/selectors/ProfilePageSelector.js

```diff
@@ -69,26 +72,9 @@ export const formErrorSelector = createSelector(
 
 export const editableFormSelector = createSelector(
   editableFormModeSelector,
-  profileAccountSelector,
-  profileVisibilitySelector,
-  formIdSelector,
   formErrorSelector,
   saveStateSelector,
-  accountDraftsFieldSelector,
-  visibilityDraftsFieldSelector,
-  (
-    editMode,
-    account,
-    visibility,
-    formId,
-    error,
-    saveState,
-    accountDraftsField,
-    visibilityDraftsField,
-  ) => ({
-    value: accountDraftsField || account[formId] || '',
-    committedValue: account[formId] || '',
-    visibility: visibilityDraftsField || visibility[formId] || 'private',
+  (editMode, error, saveState) => ({
```

Comment:
In the process of removing these last three pieces of data for the forms, then we can disconnect them again.  It was well intentioned, but created weird asymmetry in the data flow.